source/rules/no-ignored-subscription.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license Use of this source code is governed by an MIT-style license that
+ * can be found in the LICENSE file at https://github.com/cartant/eslint-plugin-rxjs
+ */
+
+import { Rule } from "eslint";
+import * as es from "estree";
+import * as ts from "typescript";
+import { couldBeType } from "tsutils-etc";
+import { getTypeCheckerAndNodeMap } from "../utils";
+
+const rule: Rule.RuleModule = {
+  meta: {
+    docs: {
+      category: "RxJS",
+      description: "Disallows ignoring the subscription returned by subscribe.",
```

Comment:
The rule name and implementation are mismatched. The implementation is that of the [`no-ignored-subscribe`](https://github.com/cartant/rxjs-tslint-rules/blob/master/source/rules/rxjsNoIgnoredSubscribeRule.ts) rule.

Reply:
🤦‍♂️ thanks for noticing this